src/main/java/blackjack/controller/Game.java

```diff
@@ -0,0 +1,117 @@
+package blackjack.controller;
+
+import blackjack.domain.Dealer;
+import blackjack.domain.Player;
+import blackjack.domain.Players;
+import blackjack.domain.Result;
+import blackjack.domain.ResultStatus;
+import blackjack.view.InputView;
+import blackjack.view.OutputView;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.stream.Collectors;
+
+public class Game {
+
+    private static final int BLACKJACK_SCORE = 21;
+    private final Dealer dealer;
+    private final Players players;
+
+
+    public Game(Dealer dealer, Players players) {
+        this.dealer = dealer;
+        this.players = players;
+    }
+
+    public void play() {
+        initializeHands();
+
+        List<Player> players = this.players.getPlayers();
+        OutputView.printInitialHands(dealer, players);
+
+        playerTurn(players, dealer);
+        dealerTurn(dealer);
+
+        OutputView.printHandsWithScore(dealer, players);
+        OutputView.printResult(calculateResult(), dealer);
+    }
+
+    private void initializeHands() {
+        dealer.shuffleDeck();
+        initializeHand(dealer);
+        for (Player player : players.getPlayers()) {
+            initializeHand(player);
+        }
+    }
+
+    private void initializeHand(Player player) {
+        player.initializeHand(dealer.draw(), dealer.draw());
+    }
+
+    private Result calculateResult() {
+        int dealerScore = dealer.calculate();
+        Map<Player, Integer> playersScore = players.calculate();
+
+        return new Result(playersScore.entrySet().stream()
+                .collect(Collectors.toMap(
+                        Entry::getKey,
+                        player -> match(dealerScore, player.getValue()),
+                        (previous, next) -> next,
+                        LinkedHashMap::new)));
+    }
+
+    private ResultStatus match(int dealerScore, int playerScore) {
+        boolean isDealerBust = dealerScore > BLACKJACK_SCORE;
+        boolean isPlayerBust = playerScore > BLACKJACK_SCORE;
+        if (isDealerBust && isPlayerBust) {
+            return ResultStatus.DRAW;
+        }
+        if (isDealerBust) {
+            return ResultStatus.WIN;
+        }
+        if (isPlayerBust) {
+            return ResultStatus.LOSE;
+        }
+        final int gap = dealerScore - playerScore;
+        return matchWhenDealerAlive(gap);
+    }
+
+    private static ResultStatus matchWhenDealerAlive(int gap) {
```

Comment:
```suggestion
    private ResultStatus matchWhenDealerAlive(int gap) {
```
메서드 분리할 때 static 이 포함되죠? 😂
이거 method 분리 시도했을 때 옆에 설정 버튼 누르고 체크박스 제거하면 default 로 static 안 붙게 할 수 있어요 :
<img width="334" alt="Screenshot 2024-03-13 at 12 51 59 AM" src="https://github.com/woowacourse/java-blackjack/assets/48251668/bf9c5839-cc34-43a0-8b21-09dbecb49a32">

Reply:
저렇게 생성된것도 모르고있었네요... 설정 팁 감사합니다🙏